libs/labelbox/src/labelbox/schema/task.py

```diff
@@ -214,3 +227,188 @@ def get_task(client, task_id):
         task: Task = tasks[0]
         task._user = user
         return task
+
+
+class DataUpsertTask(Task):
+    """
+    Task class for data row upsert operations
+    """
+    __max_donwload_size: Final = MAX_DATAROW_PER_API_OPERATION
```

Comment:
nit: Constants are typically in UPPER_CASE (at the very least we should be consistent).

Reply:
Also, `donwload` should be `download`

Reply:
this is code from Attila, I will update